proto/lbm/token/v1/tx.proto

```diff
@@ -402,14 +402,14 @@ message MsgBurnFromResponse {}
 //   - `changes` is empty.
 //
 // Signer: `owner`
-message MsgModify {
+message MsgModifyContract {
   // contract id associated with the contract.
   string contract_id = 1;
   // the address of the grantee which must have modify permission.
-  string owner = 2;
+  string operator = 2;
   // changes to apply.
```

Comment:
Could you write possible attribute keys in the comment, so that client developers accurately know them?

`message MsgModify` in `collection` has such a comment.
Please refer to https://github.com/line/lbm-sdk/blob/v0.46.0/proto/lbm/collection/v1/tx.proto#L603-L605